### PR TITLE
docs: Added v3 API reference.

### PR DESCRIPTION
**Pull request by mk46**
Signed-off-by: Manish Kumar <manish.kumar1@india.nec.com>
 
Added v3 API reference in [rocketmq_proxy_filter](https://github.com/envoyproxy/envoy/blob/main/docs/root/configuration/listeners/network_filters/rocketmq_proxy_filter.rst) docs.

### CODE REVIEW

File: docs/root/configuration/listeners/network_filters/rocketmq_proxy_filter.rst [hunk 1]
```diff
@@ -3,6 +3,9 @@
 RocketMQ proxy
 ==============
 
+* :ref:`v3 API reference <envoy_v3_api_msg_extensions.filters.network.rocketmq_proxy.v3.RocketmqProxy>`
+* This filter should be configured with the name *envoy.filters.network.rocketmq_proxy*.
```
Comment (phlax) [inline docs/root/configuration/listeners/network_filters/rocketmq_proxy_filter.rst:7]: this should be a literal - ie ` ``envoy.filters.network.rocketmq_proxy`` `

**Review by phlax — CHANGES_REQUESTED** [review]

File: docs/root/configuration/listeners/network_filters/rocketmq_proxy_filter.rst (hunk 1 above)
Comment (mk46) [reply docs/root/configuration/listeners/network_filters/rocketmq_proxy_filter.rst]: Done.

**Review by phlax — APPROVED** [review]
lgtm thanks @mk46

**Pull request merged**